korangar/src/graphics/passes/point_shadow/entity.rs

```diff
@@ -28,11 +28,11 @@ struct InstanceData {
     texture_size: [f32; 2],
     extra_depth_offset: f32,
     depth_offset: f32,
-    angle: f32,
     curvature: f32,
     mirror: u32,
     texture_index: i32,
-    padding: [u32; 2],
+    padding: u32,
+    padding_2: [u32; 2],
```

Comment:
Can you please simplify the padding here too?

Reply:
Opsss, forget to change... Good catch!